content/sips/sip-420.md

```diff
@@ -0,0 +1,105 @@
+
+## Motivation
+
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.-->
+
+No inflation and mediocre fees have created minimal incentive for SNX staking, even with the lower risk in the current system. Staking rates are at multi-year lows, and a large minority of SNX is unstaked and sitting idle on exchanges. This proposal removes the need for active individual staking and creates a capital-efficient structure that significantly enhances incentives. It also helps rectify past debt inflation issues by ensuring only willing participants remain in the protocol’s debt pool.
+
+## Specification
+
+<!--There are five sections in the Specification:
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values (Via SCCP)
+-->
+
+### Overview
+
+<!--This is a high level overview of how the SIP will solve the problem.-->
+
+1. **Create a delegated staking contract**  
+   - Stakers deposit their existing debt positions into the new pool.
+2. **Raise the current c-ratio to 1000%**  
+   - Discourage solo staking and encourage delegation to the pool.
+3. **Set the issuance ratio for the delegated pool to 200%**  
+   - Allow more efficient capital deployment without risk of bad debtors.
+4. **Transfer each debt position (but not collateral) over 12 months**  
+   - Gradual transition ensures minimal disruption to existing stakers.
+5. **Use the sUSD from the delegated pool to mint sUSDe via Ethena**  
+   - Leverage Ethena to generate yield.
+6. **Transfer the treasury staking collateral to other stakers over 2-3 years**  
+   - Redistribute collateral systematically and fairly among participants.
+7. **Incentivize a liquidity pool for sUSD/USDe**  
+   - Encourage stable trading between these two synthetic assets.
+8. **Allocate 250k SNX as a bonus**  
+   - Reward core contributors who implement this proposal, at the council’s discretion.
+
+### Rationale
+
+<!--This is where you explain the reasoning behind how you propose to solve the problem.-->
+
+The rationale behind this delegated staking model is twofold:  
+1. **Capital Efficiency & Reduced Risk**  
+   - By consolidating debt into a single protocol-owned pool, we remove the variability of individual staker defaults or liquidation risks, allowing for a lower issuance ratio.  
+2. **Enhanced Staking Incentives**  
+   - Transitioning to a delegated model removes the problem of low staking rates. The newly minted sUSDe and the liquidity incentives and debt jubilee make staking more lucrative, driving higher participation. This also addresses historical debt inflation.
+
+### Technical Specification
+
+<!--The technical specification should outline the public API of the changes proposed.-->
+
+A detailed technical design and contract specification will follow upon community feedback. It will likely involve:  
```

Comment:
FYI we actually explored technical implementation of this which would only require users who have migrated to v3 to change the pools they are connected to. I can go ahead and supply this proposal here so it doesn't have to be supplied separately if that is desired.

 This would handle all the issues mentioned below by reusing existing mechanisms.

Reply:
Yeah, I would appreciate it if you can submit a PR to add that information.